web/src/components/userpage/event-review-card.js

```diff
@@ -0,0 +1,104 @@
+import {
+  Card, 
+  CardContent, 
+  Grid, 
+  Typography
+} from '@material-ui/core';
+import EventReview from './event-review';
+import EventSchedule from './event-schedule';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {withRouter} from 'react-router-dom';
+
+class EventReviewCard extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  // Render Schedule object list into a EventSchedule component list.
+  renderSchedules(schedules) {
+    // This is for displaying schedule number.
+    let index = 1;
+
+    // Map Schedule objects to EventSchedule components.
+    schedules = schedules.map((schedule) => {
+      const {id, startTime, endTime, address} = schedule;
+      return (
+        <EventSchedule 
+          key={id}
```

Comment:
other components have the first property in the first line (but here, first property is in the next line). Can we be consistent?